Splunk/ecm/splunk-lab/azuredeploy.json

```diff
@@ -0,0 +1,89 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "containerGroupName": {
+        "type": "string",
+        "defaultValue": "splunk-container-group",
+        "metadata": {
+          "description": "Container Group name."
+        }
+      }
+    },
+    "variables": {
+      "container1name": "splunk-main",
+      "container1image": "splunk/splunk:latest",
+    },
+    "resources": [
+      {
+        "name": "[parameters('containerGroupName')]",
+        "type": "Microsoft.ContainerInstance/containerGroups",
+        "apiVersion": "2018-10-01",
+        "location": "[resourceGroup().location]",
+        "properties": {
+          "containers": [
+            {
+              "name": "[variables('container1name')]",
+              "properties": {
+                "image": "[variables('container1image')]",
+                "resources": {
+                  "requests": {
+                    "cpu": 1,
+                    "memoryInGb": 1.5
+                  }
+                },
+                "ports": [
+
+                  {
+                    "port": 8000
+                  },
+                  {
+                    "port": 9997
+                  },
+                  {
+                    "port": 9998
+                  },
+                  {
+                    "port": 9999
+                  }
+                ],
+                "environmentVariables" : [
+                    {"name": "SPLUNK_START_ARGS" , "value": "--accept-license" },
+                    {"name": "SPLUNK_PASSWORD" , "value": "SplunkServer1$" }
+                ]
+              }
+            }
+           
+          ],
+          "osType": "Linux",
+          "ipAddress": {
+            "type": "Public",
+            "ports": [
+              {
+                "protocol": "tcp",
+                "port": "8000"
+              },
+              {
+                  "protocol": "tcp",
+                  "port": "9997"
+              },
+              {
+                "protocol": "tcp",
+                "port": "9998"
+            },
+            {
+                "protocol": "tcp",
+                "port": "9999"
+            }
+            ]
+          }
+        }
+      }
+    ],
+    "outputs": {
+      "containerIPv4Address": {
+        "type": "string",
+        "value": "[reference(resourceId('Microsoft.ContainerInstance/containerGroups/', parameters('containerGroupName'))).ipAddress.ip]"
+      }
+    }
+  }
```

Comment:
newline here

Reply:
Cool that we got a template.  Keep in mind that their containers can be more expensive than VMs because they are up in multiple AZs